examples/gcp/outputs.tf

```diff
@@ -6,14 +6,20 @@ output "service_account_email" {
   value = module.bitmovin_cloud_connect.service_account_email
 }
 
-output "private_key" {
-  value     = module.bitmovin_cloud_connect.private_key
-  sensitive = true
+# partially resembles JSON key file
+resource "local_file" "service_account_key_file" {
+  content = jsonencode({
+    "type" : "service_account"
+    "project_id" : local.project_id
+    "private_key" : jsondecode(base64decode(module.bitmovin_cloud_connect.private_key))["private_key"]
+    "client_email" : module.bitmovin_cloud_connect.service_account_email
+  })
+  filename = "${path.module}/${local.project_id}-${substr(jsondecode(base64decode(module.bitmovin_cloud_connect.private_key))["private_key_id"], 0, 12)}.json"
```

Comment:
This will create an JSON output with the same file name schema and the required fields: `project_id`, `private_key` and `client_email`. Tested with `serviceAccountCredentials` via https://developer.bitmovin.com/encoding/reference/postencodinginfrastructuregce